specification.md

```diff
@@ -219,6 +225,14 @@ Optional parameters
 
 Note that the OpenTracing project documents certain **["standard log keys"](https://github.com/opentracing/specification/blob/master/data_conventions.yaml)** which have prescribed semantic meanings.
 
+##### An aside: "Logging" in general, and what it means in OpenTracing
+
+"Logging" is an overloaded term in our industry; one could reasonably argue that all tracing is just a particularly organized form of logging. OpenTracing "logs" are really just key:value maps that describe a particular moment within the context of a Span.
+
+It is not forbidden but also not particularly wise to redirect general-purpose process-level logging into OpenTracing. Some of those logs may be relevant to transaction traces, but some may also pertain to process-global state which has no direct relationship to a transaction trace.
+
+The granularity of Span logs is intended to be finer than typical "info"-style logging in process-level logging frameworks. Since tracing systems usually have a smart, all-or-nothing per-trace sampling mechanism, the verbosity within a single trace can be higher than what would be appropriate for a process as a whole — especially when that process contends with high concurrency. |
+
```

Comment:
ah yes -- a vestige of a previous attempt to make this aside part of a "sidebar" (attempt was not successful!)